scrapi/processing/cassandra.py

```diff
@@ -73,6 +73,48 @@ def different(self, old, new):
 
 
 @database.register_model
+class DocumentModelV2(models.Model):
+    '''
+    Defines the schema for a metadata document in cassandra
+
+    The schema contains denormalized raw document, denormalized
+    normalized (so sorry for the terminology clash) document, and
+    a list of version IDs that refer to previous versions of this
+    metadata.
+    '''
+    __table_name__ = 'documents_v2'
```

Comment:
You may want to change the table name and possibly even the model name to something like documents correctly clustered/ordered. v2 sounds like the storage schema may have changed.
And later on if you decide to remove the DocumentModel and replace it with this one the name will make a bit more sense.